apps/frontend/src/pages/people/index.tsx

```diff
@@ -17,17 +28,17 @@ const PeoplePage: React.FC = () => {
   useEffect(() => {
     const fetchConnections = async () => {
       const user = await storage.getUser();
-      if (!user) {
+      const unregisteredUser = await getUnregisteredUser();
+      if (!user && !unregisteredUser) {
         return;
```

Comment:
shouldn't we boot them off this page in this case?

Reply:
I now boot, addressed